format-volume/tasks/main.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Create local directory for mount point.
+  file:
+    path: "{{ mount_point }}"
+    state: directory
+- name: Make primary partition
+  parted:
+    device: "{{ device_id }}"
+    number: 1
+    label: gpt
+    part_type: primary
+    state: present
+    part_start: 0%
+    part_end: 100%
+- name: Create a ext4 format filesystem.
+  filesystem:
+    fstype: ext4
+    dev: "{{ device_id }}1"
+- name: Get UUID for device.
+  command: blkid -s UUID -o value  "{{ device_id }}1"
+  register: disk_blkid
+  changed_when: False
+- name: Mount device by UUID.
+  mount:
+    # disabling backup for now since it requires ansible 2.5 and I only have 2.4
+#    backup: yes
```

Comment:
Noticed this commented out `backup` setting. We could remove this and create an issue to look into moving to ansible 2.5+?